Framework/DataHandling/src/SaveDiffCal.cpp

```diff
@@ -120,6 +148,22 @@ void SaveDiffCal::writeIntFieldFromTable(H5::Group &group, const std::string &na
   H5Util::writeArray1D(group, name, data);
 }
 
+void SaveDiffCal::writeDetIdsfromSVWS(H5::Group &group, const std::string &name,
```

Comment:
What does abbreviation SV mean?

Reply:
SVWS = `SingleValueWorkspace`. It is the parent of `GroupingWorkspace` and `MaskingWorkspace`

Reply:
Sorry, but such class doesn't exist. I found a method `CreateSingleValuedWorkspace`. Do you mind adding a comment saying that SVWS means a _workspace with a single value per pixel_?  This would help new programmers.

Reply:
I'll add the comment, but it is actually `Mantid::DataObjects::WorkspaceSingleValue`